introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -26,7 +26,7 @@
     "---\n",
```

Comment:
<p>Minor:</p><ol><li>s/sagemaker sdk/SageMaker SDK</li><li>s/apis/APIs</li></ol><p> </p><p><code>pip</code> may be unavailable in network-isolated mode. Is there a better way to make the notebook compatible? The code in Appendix downloads a dataset from the internet which won't work either, not sure how big of an issue that is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The code work as is without pip install right now. So, I can delete the comment if required (though prefer not to).</p><p> </p><p>Overall, I feel this may be a generic problem: packages get updated which breaks the notebook. Not sure if there is any solution other than installing a particular version.</p>

Reply:
<p>Appendix is not necessarily supposed to work in Isolation mode and is not required for the notebook (before appendix, both training and inference) to run.</p>